cmd/pmp/src/lib.rs

```diff
@@ -126,8 +126,8 @@ fn pmpcmd(context: &mut humility::ExecutionContext) -> Result<()> {
             Mode::NAPOT => println!(
                 "pmpaddr{:02} {:#10x} - {:#10x} {:#7x} {}{}{}{:<2} {:#5?}",
                 i,
-                addr,
-                addr + (size.unwrap().get()) - 1,
+                addr.unwrap(),
```

Comment:
I think it would be helpful to have a note in the commit message that explains why the lib.rs file changed.